db/migrate/20201104003820_create_trips.rb

```diff
@@ -0,0 +1,13 @@
+class CreateTrips < ActiveRecord::Migration[6.0]
+  def change
+    create_table :trips do |t|
+      t.string :driver_id
+      t.string :passenger_id
+      t.string :date
+      t.string :rating
+      t.string :cost
```

Comment:
These should have been numbers:

```suggestion
      t.integer :rating
      t.float :cost
```